src/app/cli/src/cli_commands.rs

```diff
@@ -218,10 +215,7 @@ pub fn get_command(
             } else {
                 odf_server::AccessTokenStoreScope::Workspace
             },
-            // TODO: improve URL parser
-            submatches
-                .get_one::<String>("server")
-                .map(|s| Url::parse(s).unwrap()),
+            parse_server_url(submatches.get_one::<String>("server"))?,
```

Comment:
Do you need to specify `::<String>` here? I'd expect that type inference should work fine in this simple case.

Reply:
`String` was not required, but `Url` is now required